components/TestSubmissionsList/TestSubmissionsList.tsx

```diff
@@ -2,17 +2,39 @@
 import scss from './TestSubmissionsList.module.scss';
 import {TestSubmissionsResults} from "../TestSubmissionsResults/TestSubmissionsResults";
 
+interface TestSubmissionsListProps {
+    tests: string[];
+}
 
-export function TestSubmissionsList(): JSX.Element {
-    
-    //TODO: getTestSubmissions result from API
-    
-    const tests: string [] = ["Test 1", "Test 2", "Test 3"];
-    
+export function TestSubmissionsList(props: TestSubmissionsListProps): JSX.Element {
+
+    const testHolder: string [] = ["Test 1", "Test 2", "Test 3"];
     return (
         <ul className={scss.list}>
             {
-                tests.map(test => <TestSubmissionsResults key={tests.indexOf(test)} test={test}/>)
+                props.tests.length > 0 ?
+                    (props.tests.map(test => {
+                            const testParsed = JSON.parse(JSON.stringify(test));
```

Comment:
This looks fun! :) 

Its probably cleaner to use TypeScript to do this for us though!
you could instead have 

```ts
interface Test {
    testId: number;
    testResult: string;
    testAnswer: string;
}

interface TestSubmissionsListProps {
    tests: Test[];
}
```

Reply:
tried to use a test[] interface but struggling to recast string[] type in to test[] further up the line...

Reply:
oh interesting! :) 

I don't _think_ it should ever be a `string[]` - the thing we get from the API should always be a `Test[]`, and then it _ought_ to be fine?

Reply:
ok, I think this push should fix that! thanks for the help!